diff --git a/android/build.gradle b/android/build.gradle
index f324b9a9f3c3b3cf28eaac7fa09471592212bf3f..929f1b678ddaa1e78bb9c2c48d3777a303f81db3 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -8,7 +8,7 @@ buildscript {
 
     dependencies {
         classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version")
-        classpath("com.android.tools.build:gradle:7.3.1")
+        classpath("com.android.tools.build:gradle:8.11.0")
         classpath("com.diffplug.spotless:spotless-plugin-gradle:6.11.0")
     }
 }
